vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -30,6 +30,123 @@
     fused_moe_pallas = None  # type: ignore
 logger = init_logger(__name__)
 
+import os
+VLLM_REQUANT_FP8_INC = os.getenv("VLLM_REQUANT_FP8_INC", "0") in ["1", "true"]
+
+# ==-------------------------------------------------------------------------==
+# VLLM-HPU-EXT PATCH Start
```

Comment:
We have https://github.com/HabanaAI/vllm-hpu-extension/tree/deepseek_r1 now, can we move this to there?

Reply:
Thanks! I have temporarily moved all HPU extension-related code into `vllm_ext_patch.py` to ignore updating the extension commit. All extension-related code can be migrated to the extension repo at once later.
cc @xuechendi